assets/app/styles/_core.less

```diff
@@ -331,6 +331,96 @@ code {
   overflow-x: auto; // remove when we update to latest patternfly with twbs 3.3.4
 }
 
+.table-events {
+  background-color: #fff;
+  > tbody {
+    + tbody {
+      border-top-width: 1px;
+    }
+  }
+  tr {
+    th,td {
+      .column-combine {
+        display: none;
+      }
+    }
+    @media (min-width: @screen-lg-min) {
+      td[data-title="Name"] {
+        white-space: nowrap;
+      }
+    }
+  }
+  // Combine event reason and message into one column between 768 and 1100, so that table doesn't cause horizontal scroll and message is readable
+  @media (min-width: @screen-sm-min) and (max-width: @screen-md-max) {
+    > thead {
+      > tr {
+        > th {
+          &:nth-child(4) {
+            display: none;
+          }
+          &:nth-child(5) {
+            .column-combine {
+              display: inline;
+            }
+          }
+        }
+      }
+    }
+    > tbody {
+      > tr {
+        > td {
+          &:nth-child(4) {
+            display: none;
+          }
+          &:nth-child(5) {
+            .column-combine {
+              display: block;
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
```

Comment:
When > 768px the table display switches to a two column layout, with the table header on the left and data on the right.